packages/components/src/components/EmptyContent/EmptyContent.tsx

```diff
@@ -0,0 +1,43 @@
+import { ddsBaseTokens } from '@norges-domstoler/dds-design-tokens';
+import { HTMLAttributes } from 'react';
+import styled from 'styled-components';
+import { Typography } from '../Typography';
+
+const { colors } = ddsBaseTokens;
+
+const StyledEmptyContent = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 10rem;
+  height: 100%;
+  width: 100%;
+  background-color: ${colors.DdsColorNeutralsGray1};
+  padding: 1.5rem;
+`;
+
+const StyledEmptyContentText = styled.div`
+  max-width: 70ch;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
```

Comment:
Her kan vi også bruke design-tokens
```suggestion
  gap: ${spacing.SizesDdsSpacingLayoutX1};
```